ci-operator/step-registry/set-sample-operator/disconnected/set-sample-operator-disconnected-commands.sh

```diff
@@ -130,12 +130,12 @@ function check_mirror_registry () {
 }
 
 set_proxy
-run_command "oc get config.samples.operator.openshift.io cluster"; ret=$?
+ret=0
+run_command "oc get config.samples.operator.openshift.io cluster" || ret=$?
```

Comment:
From failed job, I didn't see any echo message from `if` or `else` block.
It seems that failure is in function run_command when running `run_command "oc get config.samples.operator.openshift.io cluster"`, the update could not fix the issue in failed job.

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/32634/rehearse-32634-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.12-installer-rehearse-aws/1575797602731954176/artifacts/installer-rehearse-aws/set-sample-operator-disconnected/build-log.txt

Reply:
Hi @jinyunma , I guess you checked the wrong job, the above job ran on Sep. 30. The `||` will address this issue.
<img width="1484" alt="image" src="https://user-images.githubusercontent.com/15416633/194788240-e438dcbc-2034-4c29-8dca-e7d9cd68e3a4.png">

Reply:
get it, thanks for checking.